bugbug/model.py

```diff
@@ -239,7 +239,7 @@ def print_feature_importances(
                 predicted_class_index = class_probabilities.argmax(axis=-1)[0]
             else:
                 predicted_class_index = 0
-
+            feature_names = self.get_human_readable_feature_names()
```

Comment:
You are defining this in the `if` branch, which means it won't be available in the `else` where it is used...